honeycomb/hive.py

```diff
@@ -15,15 +21,25 @@ def run_lake_query(query, engine='hive'):
             The querying engine to run the query through
             Use 'presto' for faster, ad-hoc/experimental querie
             Use 'hive' for slower but more robust queries
+        has_complex_cols_and_joins (bool, default False):
+            Whether the query involves both complex cols and joins. Indicating
+            this beforehand will save query time later, as it allows for
+            avoiding error handling associated with running a query like that
+            without special treatment. Caused by a hive bug
     """
+    if has_complex_cols_and_joins:
```

Comment:
Would love suggestions on a shorter/better name for this variable!

Reply:
`has_comples` or just `complex`? Your docs cover the meaning of this variable pretty well I feel.

Reply:
Or maybe just `complex_join`? Because complex columns on their own work fine, it's only with joining that they cause issues

Reply:
i like `complex_join`, super clear to me what's going on there without being too verbose

Reply:
+1